space-engineers-genericconfig.json

```diff
@@ -1,2046 +1,2443 @@
+      "FieldName": "EnvironmentDamageMultiplier",
+      "InputType": "number",
+      "IsFlagArgument": false,
+      "ParamFieldName": "/MyConfigDedicated/SessionSettings/EnvironmentDamageMultiplier | /MyObjectBuilder_WorldConfiguration/Settings/EnvironmentDamageMultiplier",
+      "IncludeInCommandLine": false,
+      "DefaultValue": "1",
+      "EnumValues": {}
+    },
+    {
+      "DisplayName": "Enable Family Sharing",
+      "Category": "SE - Players",
+      "Description": "Enables shared accounts to join multiplayer games.",
+      "Keywords": "Family,Sharing",
+      "FieldName": "FamilySharing",
+      "InputType": "checkbox",
+      "IsFlagArgument": false,
+      "ParamFieldName": "/MyConfigDedicated/SessionSettings/FamilySharing | /MyObjectBuilder_WorldConfiguration/Settings/FamilySharing",
+      "IncludeInCommandLine": false,
+      "DefaultValue": "true",
+      "EnumValues": {
+        "True": "true",
+        "False": "false"
+      }
+    },
+    {
+      "DisplayName": "Max Players",
+      "Category": "SE - Players",
+      "Description": "The maximum number of connected players. (2 - 64)",
+      "Keywords": "Max,Players,users",
+      "FieldName": "$MaxUsers",
+      "InputType": "number",
+      "IsFlagArgument": false,
+      "ParamFieldName": "/MyConfigDedicated/SessionSettings/MaxPlayers | /MyObjectBuilder_WorldConfiguration/Settings/MaxPlayers",
+      "IncludeInCommandLine": false,
+      "DefaultValue": "4",
+      "Placeholder": "4",
+      "Suffix": "Players",
+      "EnumValues": {}
+    },
+    {
+      "DisplayName": "Optimal Respawn Distance",
+      "Category": "SE - Players",
+      "Description": "Sets optimal distance in meters the game should take into consideration when spawning new player near others.",
+      "Keywords": "Optimal,Respawn,Distance",
+      "FieldName": "OptimalSpawnDistance",
+      "InputType": "number",
+      "IsFlagArgument": false,
+      "ParamFieldName": "/MyConfigDedicated/SessionSettings/OptimalSpawnDistance | /MyObjectBuilder_WorldConfiguration/Settings/OptimalSpawnDistance",
+      "IncludeInCommandLine": false,
+      "DefaultValue": "16000",
+      "Placeholder": "16000",
+      "Suffix": "Meters",
+      "EnumValues": {}
+    },
+    {
+      "DisplayName": "Permanent Death",
+      "Category": "SE - Players",
+      "Description": "Enables permanent death.",
+      "Keywords": "permanent,death",
+      "FieldName": "PermanentDeath",
+      "InputType": "checkbox",
+      "IsFlagArgument": false,
+      "ParamFieldName": "/MyConfigDedicated/SessionSettings/PermanentDeath | /MyObjectBuilder_WorldConfiguration/Settings/PermanentDeath",
+      "IncludeInCommandLine": false,
+      "DefaultValue": "false",
+      "EnumValues": {
+        "True": "true",
+        "False": "false"
+      }
+    },
+    {
+      "DisplayName": "Reset Ownership",
+      "Category": "SE - Players",
+      "Description": "Reset Ownership of all blocks to nobody.",
```

Comment:
Lower case o?

Reply:
No it's definitely capitalized. Oh, it was a rhetorical question...